tests/rhui3_tests/test_client_management.py

```diff
@@ -56,7 +93,56 @@ def test_05_create_cli_rpm():
     Expect.enter(connection, 'q')
     Expect.ping_pong(connection, "test -f /root/test_cli_rpm-3.0/build/RPMS/noarch/test_cli_rpm-3.0-1.noarch.rpm && echo SUCCESS", "[^ ]SUCCESS")
 
-def test_06_create_docker_cli_rpm():
+def test_08_remove_rhui_conf_rpm():
+    '''
+       remove rhui configuration rpm from client
+    '''
+    Util.remove_conf_rpm(cli)
```

Comment:
How about this change, since it's all about the `rh-amazon-rhui-client` package:

```
-    def remove_conf_rpm(connection):
+    def remove_amazon_rhui_conf_rpm(connection):
         '''
-        Remove RHUI configuration rpm from instance (which owns /etc/yum/pluginconf.d/rhui-lb.conf file)
+        Remove Amazon RHUI configuration rpm from instance (which owns /etc/yum/pluginconf.d/rhui-lb.conf file)

```